dotnet/src/support/Events/FindElementEventArgs.cs

```diff
@@ -1,4 +1,4 @@
-﻿// <copyright file="FindElementEventArgs.cs" company="WebDriver Committers">
+// <copyright file="FindElementEventArgs.cs" company="WebDriver Committers">
```

Comment:
what is different in these lines?

Reply:
Invisible character:

![image](https://github.com/SeleniumHQ/selenium/assets/22616990/abe380c4-4489-4f82-bd3a-fd059c135d1b)